ci-operator/step-registry/telco5g/sno-cnf/telco5g-sno-cnf-workflow.yaml

```diff
@@ -5,11 +5,14 @@ workflow:
     - ref: telco5g-jobs-router
     - ref: telco5g-sno-setup
     test:
-    - ref: telco5g-cnf-tests
+    # - ref: telco5g-cnf-tests
+    - ref: telco5g-cnf-rt-tests
+    env:
```

Comment:
move this to after 'steps'  it's general to workflow, and not specific to 'ref'.
shoudn't it be 'true/false' without quotes?

Reply:
Received error when using boolean.
`could not determine changed registry steps: could not load step registry: failed to load registry file /var/tmp/gitrepo870627500/ci-operator/step-registry/telco5g/cnf/rt-tests/telco5g-cnf-rt-tests-ref.yaml: error unmarshaling JSON: while decoding JSON: json: cannot unmarshal bool into Go struct field StepParameter.ref.env.default of type string`